spec/jobs/send_webhook_job_spec.rb

```diff
@@ -200,7 +225,7 @@
 
   context 'with not implemented webhook type' do
     it 'raises a NotImplementedError' do
-      expect { described_class.perform_now(:subscription, invoice) }
+      expect { subject.perform_now(:subscription, invoice) }
```

Comment:
RSpec/NamedSubject: Name your test subject if you need to reference it explicitly. (https://rspec.rubystyle.guide/#use-subject, https://www.rubydoc.info/gems/rubocop-rspec/RuboCop/Cop/RSpec/NamedSubject)